db/migrate/20210803170531_change_unique_index_for_schedules.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeUniqueIndexForSchedules < ActiveRecord::Migration[6.1]
+  def up
+    remove_index :schedules, name: "unique_child_schedules"
+    add_index :schedules, %i[effective_on child_id weekday], unique: true, name: :unique_child_schedules
+  end
+  def down
```

Comment:
~Your `down` is doing super similar things to the `up`. 🤔 I would expect the `down` to do the opposites?~

NVM.